### PR TITLE
[js] Hallo

### PR DESCRIPTION
**Pull request by Teodor1990Ravineala**

### CODE REVIEW

File: SomeFile.js [hunk 1]
```diff
@@ -0,0 +1 @@
+//change from Teodor
```
Comment (rasnaut) [inline SomeFile.js:1]: Word change need to be from capital letter

**Review by rasnaut — CHANGES_REQUESTED** [review]
Please change according comments

File: SomeFile.js (hunk 1 above)
Comment (Teodor1990Ravineala) [reply SomeFile.js]: Fixed

**Review by rasnaut — APPROVED** [review]

**Review by mvadyan — APPROVED** [review]

**Review by ValeryMukhin1711 — APPROVED** [review]
ok

**Pull request merged**